_studio/shared/include/mfx_ext_buffers.h

```diff
@@ -79,6 +79,7 @@ typedef struct {
 
     mfxU16 DisablePSubMBPartition;  // tri-state, default depends on Profile and Level
     mfxU16 DisableBSubMBPartition;  // tri-state, default depends on Profile and Level
+    mfxU16 Transform8x8Mode;        // tri-state: 0, MFX_CODINGOPTION_OFF, MFX_CODINGOPTION_ON
```

Comment:
Please don't just insert in the middle. It will break binary compatibility with older versions of apps/tools which use this buffer. Please replace reserved field instead.